packages/react-components/src/components/grid.tsx

```diff
@@ -69,59 +67,23 @@ class Grid extends PureComponent<Props, State> {
         return null
     }
 
-    setBricks() {
-        const { columns, gutter } = this.props
-        // bricks
-        this.bricks = Bricks({
-            container: this.el!,
-            packed: `data-packed-${columns}`,
-            sizes: [{ columns, gutter }],
-        })
-    }
-
     componentDidMount() {
-        this.setBricks()
         this.onFetch()
     }
 
     componentWillUnmount() {
         this.unmounted = true
     }
 
-    componentDidUpdate(prevProps: Props, prevState: State) {
-        const { gifs } = this.state
-        const { gifWidth } = this.state
-
-        const numberOfOldGifs = prevState.gifs.length
-        const numberOfNewGifs = gifs.length
-
-        if (prevState.gifWidth !== gifWidth && numberOfOldGifs > 0) {
-            const { columns } = this.props
-            if (columns !== prevProps.columns) {
```

Comment:
Moving the bricks component to React simplified this, no need to pack and update, just render